planetiler-core/src/main/java/com/onthegomap/planetiler/reader/GeoPackageReader.java

```diff
@@ -39,9 +44,14 @@ public class GeoPackageReader extends SimpleReader<SimpleFeature> {
   private final GeoPackage geoPackage;
   private final MathTransform coordinateTransform;
 
-  GeoPackageReader(String sourceProjection, String sourceName, Path input, Path tmpDir, boolean keepUnzipped) {
+  private final Envelope bounds;
+
+  GeoPackageReader(String sourceProjection, String sourceName, Path input, Path tmpDir, boolean keepUnzipped,
+    Envelope bounds) {
```

Comment:
For consistency with ShapefileReader should this take `Bounds` class?

Reply:
Changed to `Bounds`